### DISCUSSION TITLE
hola

### DISCUSSION THREAD

**Discussion by marianasalces** [atralice/Curso.Prep.Henry#81] [General]
esto seria el slack? en el video no dicen en cual pestana entrar para ingresar al slack, en el ideo ya estan dentro. espero sea este jaja. help!

**Comment by AnaLiaR** [comment]
Hola. no, este no es el Slack. Yo tampoco sé como hacer para conectar a Slack, pero seguro no es este ;)

**Reply by marianasalces** [reply]
hola ana . Ya lo "descubri" jaja . Hay que escargar la aplicacion , te dejo el link del que la descargue, espero sea compatible con tu pc. Exitos. Saludos! https://app.slack.com/client/T0278NV6NAD/C026W3JPEHY
